platform/linux-generic/odp_system_info.c

```diff
@@ -527,6 +527,18 @@ int odp_cpu_count(void)
 	return odp_global_ro.system_info.cpu_count;
 }
 
+int odp_system_info(odp_system_info_t *info)
+{
+	system_info_t *sys_info = &odp_global_ro.system_info;
+
+	memset(info, 0, sizeof(odp_system_info_t));
+
+	info->cpu_arch   = sys_info->cpu_arch;
+	info->cpu_isa_sw = sys_info->cpu_isa_sw;
```

Comment:
Should these values be initialized also for the default platform? E.g. `_odp_dummy_cpuinfo()`

Reply:
Initialized sys_info->cpu_arch to ODP_CPU_ARCH_UNKNOWN. sys_info->cpu_isa_sw is memset to zero and should not be used when cpu_arch is unknown.